packages/widget/src/components/network-selector/network-selector.ts

```diff
@@ -16,70 +17,91 @@ type Direction = (typeof Directions)[keyof typeof Directions];
 export class NetworkSelector extends LitElement {
   static styles = styles;
 
-  @property({
-    type: Boolean
-  })
-  disabled = false;
+  @state()
+  private _isDropdownOpen = false;
```

Comment:
if we use the accessor private, does it make sense to use `_something` notation? Wdyt?

Reply:
it will work without underscore, I think it's just an old naming convention to indicate the private method in the class. But we can agree on the way how to properly write it.

Reply:
we have some of them in the others components as well, but there are not private per se, just "internal" I guess. I'm not against it, but we should have some agreement. For instance `_onResourceSelected` on amount component is only being used inside the component, so it could make sense either to add `private` and mantain the naming, or add private and remove the underscore.

Reply:
Let's remove then private